modules/caddyhttp/fileserver/browsetplcontext.go

```diff
@@ -52,11 +52,21 @@ func (fsrv *FileServer) directoryListing(files []os.FileInfo, canGoUp bool, root
 			fileCount++
 		}
 
+		isSymlink := isSymlink(f)
```

Comment:
Though technically OK, I don't think shadowing a function name with a local variable name is a good idea, as it can easily be confusing. When I initially looked at line 67 below, I was trying to figure out why passing a function value was working, then I finally saw this, that it was a local variable in this particular spot. Let's rename the variable to something different.

Reply:
I suggest `fileIsSymlink`

Reply:
I see, thanks for the suggestion!
I fix this.